src/web/mjs/connectors/Allanimesite.mjs

```diff
@@ -37,40 +35,71 @@ export default class Allanimesite extends Connector {
         for(let page = 1, run = true; run; page++) {
             const mangas = await this._getMangasFromPage(page);
             await this.wait(this.config.throttle.value);
-            mangas.length ? mangaList.push(...mangas) : run = false;
+            if(mangas.length == 0) {
+                run = false;
+            } else if (mangaList.length == 0 || mangas[mangas.length - 1].id !== mangaList[mangaList.length - 1].id) {
+                mangaList.push(...mangas);
+            } else {
+                run = false;
+            }
         }
         return mangaList;
     }
     async _getMangasFromPage(page) {
-        const uri = new URL(this.path + this.varQueryMangas.replace('%PAGE%', page), this.url);
+
+        const jsonVariables = {
+            search : {
+                isManga : true,
+                allowAdult : false,

```

Comment:
Why not allow adult, i like adult stuff 😢

Reply:
Like i said in comments, apparently some variables doesnt seems to have consequences on search results.
I tried to increase manga / animes per pages, api doesnt care.

Well, gonna set adult and unknown to true since it doesnt seems to cause errors.